packages/plugins/rubygem-foreman_remote_execution/rubygem-foreman_remote_execution.spec

```diff
@@ -95,11 +96,16 @@ exit 0
 %{gem_instdir}/locale
 %{gem_instdir}/lib
 %{gem_instdir}/public
+%{gem_instdir}/webpack
```

Comment:
This and `package.json` can be excluded because they're not needed in production. I do wonder if we should create a -assets package with these files.